fixes.inc

```diff
@@ -9796,6 +9865,67 @@ native BAD_SetPlayerName(playerid, const name[]) = SetPlayerName;
 		// =====================
 		//  BEGIN: OnVehicleMod
 		// =====================
+
+
+		switch(FIXES_GetVehicleComponentType(componentid))
+		{
+			#if !FIXES_Single
+				case CARMODTYPE_STEREO:
+				{
+					setproperty(.name = FIXES_gscStereo, .value = componentid);
+					return FIXES_OnVehicleMod(playerid, vehicleid, componentid);
+				}
+				case CARMODTYPE_FRONT_BUMPER:
+				{
+					setproperty(.name = FIXES_gscFrontBumper, .value = componentid);
+					return FIXES_OnVehicleMod(playerid, vehicleid, componentid);
+				}
+				case CARMODTYPE_REAR_BUMPER:
+				{
+					setproperty(.name = FIXES_gscRearBumper, .value = componentid);
+					return FIXES_OnVehicleMod(playerid, vehicleid, componentid);
+				}
+				case CARMODTYPE_FRONT_BULLBAR:
+				{
+					setproperty(.name = FIXES_gscFrontBullBar, .value = componentid);
+					return FIXES_OnVehicleMod(playerid, vehicleid, componentid);
+				}
+				case CARMODTYPE_REAR_BULLBAR:
+				{
+					setproperty(.name = FIXES_gscRearBullbar, .value = componentid);
+					return FIXES_OnVehicleMod(playerid, vehicleid, componentid);
+				}
+			#else 
+				case CARMODTYPE_STEREO:
+				{
+					FIXES_gsStereo[vehicleid] = componentid;
+					return FIXES_OnVehicleMod(playerid, vehicleid, componentid);
+				}
+				case CARMODTYPE_FRONT_BUMPER:
+				{
+					FIXES_gsFrontBumper[vehicleid] = componentid;
+					return FIXES_OnVehicleMod(playerid, vehicleid, componentid);
+				}
+				case CARMODTYPE_REAR_BUMPER:
+				{
+					FIXES_gsRearBumper[vehicleid] = componentid;
+					return FIXES_OnVehicleMod(playerid, vehicleid, componentid);
+				}
+				case CARMODTYPE_FRONT_BULLBAR:
+				{
+					FIXES_gsFrontBullBar[vehicleid] = componentid;
+					return FIXES_OnVehicleMod(playerid, vehicleid, componentid);
+				}
+				case CARMODTYPE_REAR_BULLBAR:
+				{
+					FIXES_gsRearBullbar[vehicleid] = componentid;
+					return FIXES_OnVehicleMod(playerid, vehicleid, componentid);
```

Comment:
Should this code fall through to the later code in this callback?  Basically, should the `OnVehicleMod` fix be applied in all these cases as well?